hasura/metadata/databases/bdjuno/tables/public_akash_provider.yaml

```diff
@@ -5,6 +5,13 @@ object_relationships:
 - name: account
```

Comment:
Should add `akash_provider_inventory` to `array_relationship` 
```
array_relationships: 
- name: akash_provider_inventories
  using:
    foreign_key_constraint_on:
      column: provider_address
      table:
        name: akash_provider_inventory
        schema: public
```

Reply:
Oh I think it's `object_relationship` because 1 provider address can only get 1 inventory status. So I modified the relationship's name instead from `provider_addresses` to `akash_provider_inventories`